internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -10,7 +10,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
 	clusterv1 "open-cluster-management.io/api/cluster/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	cr_client "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
how about `k8s_client`?

Reply:
there is already a k8s_client. This is fine I think